File: OverDroch/overParser/src/main/java/com/meew/overparser/data/Hero.java
package com.meew.overparser.data;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

import java.util.List;
import java.util.Map;

@Getter
@Setter
public class Hero {
    String name;
    List<Ability> abilities;
    AbilityConnectionInfo abilityConnectionInfoMap;
}

File: OverDroch/overdroch2/app/src/main/java/com/meew/overdroch/data/hero/Hero.kt
package com.meew.overdroch.data.hero

data class Hero(
        var key: String,
        var name: String = "sample",
        var portrait: Any?,
        var role: Role?,
        var location: String = "",
        var abilities: List<Ability>?,
        var story: Story? = null
        )